### ISSUE TITLE
missing recipe: wooden gear set and iron gear set.

### ISSUE THREAD

**Issue by Grounli** [P3pp3rF1y/AncientWarfare2#477]
hi, i am using google translator to report this problem because I do not speak English . The 'wooden gear set ' and 'iron gear set ' apparently have no recipe, and there is no legal way to create it, the 'No More Recipe Conflict' does not solve the problem because there is no recipe on those objects specifically . I do not know why they have gone, and I do not know if it is a conflict with another 'mod' .

Screenshots:
http://www.mediafire.com/view/p339x9l4u1jlelu/2016-04-09_13.54.53.png
http://www.mediafire.com/view/w1nyyq37oy1znw5/2016-04-09_13.55.03.png

Greetings.

**Comment by DeathOfTime** [comment]
Having the same issue. No recipe is appearing for the wooden or iron gear set in NEI. The research table won't accept any other gear for the research. I am running a modpack called Death Of Sinks, that is on Curse. I do notice in the "config\ancientwarfare\AncientWarfare.cfg" that their is a reference on lines 354 to 358 to gears from a mod that is not in the pack. It references buildcraft|Core and I am using buildcraft. I am not for sure of the distinction. I just know that I have had a lot of issues with Buildcraft, depending on which set I install I get different issues.

This is the current config file. (line 355 to 358 is are the ones I am referring to) I have turned off steel ingot recipe. Though deleting the configs and starting up the game saw no change in the gear recipes.
http://pastebin.com/jmQ1NJAG

mods folder jar list ~ http://pastebin.com/9JnUa0DD
mod list ~ http://pastebin.com/1yPx8vEM

Edit:
Found the recipes were for crafting build craft gears and not ancient warfare gear sets. Without Buildcraft installed the will craft gear set instead. With it installed the will craft build craft gears only.  The buildcraft gears can't be used for research though.

**Comment by Fyrehanabi** [comment]
Also having issue with the iron gear set. I have buildcraft installed but to get the wooden gear set i just turned off the buildcraft wooden and iron gears in the config, this fixed the issues with the wooden gear set but the iron gear set is still showing no recipe. It shows up as an item in creative mode but not in survival.

**Comment by electromatter** [comment]
I also had this problem, a simple work-around is to add a recipe to craft the gear sets whose recipes conflict. For anyone encountering this issue in the future, here is a Minetweaker script that will add a recipe from any gear to the gear set. You may need to edit it to fit your mod pack.

```
recipes.addShapeless(<AncientWarfare:component>, [<ore:gearWood>]);
recipes.addShapeless(<AncientWarfare:component:1>, [<ore:gearIron>]);
```

**Comment by P3pp3rF1y** [comment]
Closing as this is now resolved in 1.12
